rfaas/include/rfaas/rdma_allocator.hpp

```diff
@@ -0,0 +1,38 @@
+//
+// Created by mou on 4/2/23.
+//
+
+#ifndef __RFAAS_RDMA_ALLOCATOR_HPP__
+#define __RFAAS_RDMA_ALLOCATOR_HPP__
+
+#include <cstddef>
+#include <rdmalib/buffer.hpp>
+#include <rfaas/executor.hpp>
+
+namespace rfaas {
+  template<typename T>
```

Comment:
I think the allocator should be of type `T`, and then always return `rdmalib::Buffer` object. If you look at the [C++ allocator concept](https://en.cppreference.com/w/cpp/named_req/Allocator), then I think we should use `A::pointer` as `rdmalib::Buffer`.